setup.py

```diff
@@ -82,7 +82,9 @@ def get_extras_require():
     "cmdclass": versioneer.get_cmdclass(),
     "install_requires": required,
     "extras_require": get_extras_require(),
-    "packages": find_packages(exclude=["contrib*", "docs*", "tests*", "examples*", "scripts*"]),
+    "packages": find_packages(
+        exclude=["contrib*", "docs*", "tests*", "examples*", "scripts*"]
+    ),
```

Comment:
`setup.py` PR change got through our contrib CI without being properly formated.

Forcing this change to pass CI>

Reply:
See https://github.com/great-expectations/great_expectations/pull/6756